bitnami/kafka/values.yaml

```
@@ -722,6 +722,9 @@ service:
   ## Headless service properties
   ##
   headless:
+    ## @param service.headless.publishNotReadyAddresses Indicates that any agent which deals with endpoints for this Service should disregard any indications of ready/not-ready
+    ## ref: https://manifests.io/k8s-1.21/service.spec
```

Could you point to kubernetes site documentation instead of that link ?

```suggestion
    ## ref: https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/
```